modules/apps/transfer/types/msgs_test.go

```diff
@@ -83,13 +83,13 @@ func TestMsgTransferValidation(t *testing.T) {
 }
 
 // TestMsgTransferGetSigners tests GetSigners for MsgTransfer
-func TestMsgTransferGetSigners(t *testing.T) {
+func (suite *TypesTestSuite) TestMsgTransferGetSigners() {
 	addr := sdk.AccAddress(secp256k1.GenPrivKey().PubKey().Address())
 
 	msg := types.NewMsgTransfer(validPort, validChannel, coin, addr.String(), receiver, timeoutHeight, 0, "")
-	res := msg.GetSigners()
-
-	require.Equal(t, []sdk.AccAddress{addr}, res)
+	signers, _, err := suite.chainA.GetSimApp().AppCodec().GetMsgV1Signers(msg)
```

Comment:
is it possible to refactor this to use the `moduletestutil.MakeTestEncodingConfig()` as well? :)